### DISCUSSION TITLE
Change node's text

### DISCUSSION THREAD

**Discussion by xobotyi** [ianstormtaylor/slate#4215] [Q&A]
Is there a way to directly replace node's text?
Having a text node i'm unable to change it's value via `setNode` and i cant find any other suitable method in docs.

As for now i have to create new node after current and then remove current.

**Comment by horacioh** [comment]
hey @xobotyi !
can you create a reproducible example of your issue? maybe in codesandbox would be great!

I might have a solution but I need a little bit more context on your real issue. thanks!
